docs/examples/009-subsetting.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    bbox_subset_nodes = uxds['relhum_200hPa'][0].subset.bounding_box(lon_bounds, lat_bounds, element='nodes')
</pre><p>Would it be a good idea to emphasize, or maybe show, this new  <code>bbox_subset_nodes</code> is an <code>UxDataArray</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>